### ISSUE TITLE
[FEATURE] send a reminder notification for in activity

### ISSUE THREAD

**Issue by Dun-sin** [Dun-sin/Whisper#309]
### Description

Send a browser notification If a user hasn't responded, we send a reminder notification like “don't leave your partner hanging, drop a response”

### 👀 Have you checked if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read and understood the rules in the [Contributing Guidelines](https://github.com/Dun-sin/Whisper/blob/main/CONTRIBUTING.md)

### Are you willing to work on this issue ?

None

**Comment by Chetax** [comment]
Hello @Dun-sin  ! I'm interested in solving this :D Can You Please Assign Me.

**Comment by Intekhabilyas** [comment]
@Dun-sin Please assign this to me

**Comment by Dun-sin** [comment]
> Hello @Dun-sin  ! I'm interested in solving this :D Can You Please Assign Me.

Don't know why @Intekhabilyas @Chetax are coming back to back lol. You both are fighting or? 😆

**Comment by Chetax** [comment]
Sir To Get Hands On Practicing  Through Which I get To Know Hot To Add an Reminder notification for someone has nothing does any activity from 5-10 min. Be Honest I doesn't Know Hot To do this notification In real time but i know , i will manage this by using node js .correct me if i am wrong . So @Dun-sin So Sir Please Give Me This Apportunity To Work Upon It.

**Comment by Dun-sin** [comment]
> Sir To Get Hands On Practicing  Through Which I get To Know How To Add an Reminder notification for someone has nothing does any activity from 5-10 min. Be Honest I doesn't Know How To do this notification In real time but i know , i will manage this by using node js .correct me if i am wrong . So @Dun-sin So Sir Please Give Me This Apportunity To Work Upon It. 
> 

You are already assigned to something else, please read the rules of the project , and there must be a way to do this in the client side

**Comment by satvikchachra** [comment]
May I pick this @Dun-sin ?

My approach to solve this would be to utilise "Web Notifications API" inbuilt in browsers

**Comment by Dun-sin** [comment]
> May I pick this @Dun-sin ?
> 
> My approach to solve this would be to utilise "Web Notifications API" inbuilt in browsers

sure, but i want to see how @nkBrew fixes #306, so every other related issue can follow that code standard, i will assign it to you, but please check #306 and see what we talked about, but don't work on it yet until #306 is completed.

In the meantime you can read the readme and the contributing guidelines

**Comment by Dun-sin** [comment]
@satvikchachra started this yet?

**Comment by satvikchachra** [comment]
> @satvikchachra started this yet? well if you haven't, you can now.
> 
> > Note: make the notification to send after 2 minutes of inactivity

Yes, have picked it, will complete and raise a PR on the weekend

**Comment by satvikchachra** [comment]
Hey, apologies I couldn't complete this, due to some personal emergency.
Please feel free to un-assign this ticket. 
Thank you for understanding.
